test/common/http/header_map_impl_speed_test.cc

```diff
@@ -110,7 +110,7 @@ static void headerMapImplSetInlineInteger(benchmark::State& state) {
   auto headers = Http::ResponseHeaderMapImpl::create();
   addDummyHeaders(*headers, state.range(0));
   for (auto _ : state) { // NOLINT
-    headers->setConnection(value);
+    headers->setContentLength(value);
```

Comment:
pre-existing condition, and not germane to this PR, but maybe you could leave a note if you agree or just do the change.

I think an aggressive optimizing compiler might be able to (now or someday) factor the string conversion here since value is const over all loop iterations. Probably this should be value++.